src/main/drivers/pwm_mapping.h

```diff
@@ -36,10 +36,16 @@
 #define MAX_INPUTS  8
 
 #define PWM_TIMER_MHZ 1
-#define ONESHOT125_TIMER_MHZ 4
-#define ONESHOT42_TIMER_MHZ 12
-#define MULTISHOT_TIMER_MHZ 12
-#define PWM_BRUSHED_TIMER_MHZ 24
```

Comment:
We should keep this on 24. This will give 750 steps instead of 250 an brushed PWM rate of 32kHz.